### ISSUE TITLE
[FEATURE] Integration of TTS services (Azure, Elevenlabs, others)

### ISSUE THREAD

**Issue by SikamikanikoBG** [FlowiseAI/Flowise#3316]
**Describe the feature you'd like**
Is it feasible to incorporate Eleven Labs, Azure AI STT, local STT, and other STT services into the configuration of the embedded chat to facilitate the integration of voice bots?

**Additional context**
Voice bots are rapidly gaining traction, and this feature will pave the way for seamless speech-to-speech communication. I am confident that it will bring immense value to the Flowise roadmap.

**Comment by davide445** [comment]
I'm interested in this feature for a project now, wanted to ask if there is any planned release date

**Comment by home-apps** [comment]
Indeed, 11Labs integration would give super input powers to the already powerful Flowise.

**Comment by chris409409** [comment]
@SikamikanikoBG 
I am urgently looking for a solution for the speech output (text-to-speech) for Flowise. You can find information about speech to text, but no solution for text to speech. I can't find a module for it either. Who can help me with this?
Thanks, Chris.

**Comment by howardtokka** [comment]
@chris409409 

If you are using the Embed UI provided by Flowise I guess no luck for now. Or you need to customize it.
I have developed a totally new wrapper chat UI and use flowise API to interact with my chatflows as a backend. In this case I have full control of what to display and what to pre or post process from a prediction.
